README.md

```diff
@@ -3,19 +3,65 @@ React Native Tipsi custom UI elements
 
 ## Components
 
+### `<ColoredTabs />`
+
+Customizable colored tabs.
+
+#### ColoredTabs Props
+
+| Name | Desc | Type | Default
+| --- | --- | --- | --- |
+| `selected` | id of selected tab.  | String | `undefined`
+| `onPress` | Handle press on tab action. Receive id of selected tab as argument. | Function | `undefined`
+
+
+#### ColoredTabs.Item Props
+
+| Name | Desc | Type | Default
+| --- | --- | --- | --- |
+| `name` | Text of tab. | String | `undefined`
```

Comment:
It would be nice if you rename this prop into `title` like in other components // @itsmepetrov